go-sms_inbound.js

```diff
@@ -1363,6 +1363,8 @@ go.app = function() {
             switch (user_first_word) {
                 case "STOP":
                     return self.states.create("state_find_identity");
+                case "BABY":
```

Comment:
I know for momconnect we have a whole list of words here, mostly the word baby in the supported languages. Were we only given this word to check for?